code/modules/antagonists/nuclear_operative/nuclear_operative.dm

```diff
@@ -8,6 +8,8 @@
 
 	var/static/commander_title
 	var/static/available_callsigns
+	var/list/purchased_items = list() //Used for adding a nukie's vendor purchases to crew credits. Items are tracked by whoever interacts with the vendor, so if the whole team gives their credits to the commander, the commander will have multiple entries in the crew credits!
+	var/list/uplink_items = list() // Same but for custom uplinks
```

Comment:
Same

Reply:
I picked datum/syndicate_buylist for this for similar reasons to why I used datum/materiel-- its what uplinks.dm uses (line 689 there, for instance)